.storybook/stories/forms/forms-input-states-components.stories.ts

```diff
@@ -0,0 +1,210 @@
+      </clr-select-container>
+      <clr-checkbox-container>
+        <label>Checkbox</label>
+        <clr-checkbox-wrapper>
+          <input type="checkbox" clrCheckbox name="option1" value="option1" formControlName="option1" />
+          <label>Option 1</label>
+        </clr-checkbox-wrapper>
+        <clr-control-helper>Helper Subtext</clr-control-helper>
+        <clr-control-error>State Subtext</clr-control-error>
+        <clr-control-success>State Subtext</clr-control-success>
+      </clr-checkbox-container>
+      <clr-date-container>
+        <label>Datepicker</label>
+        <input type="date" autocomplete="off" clrDate formControlName="date" name="date" />
+      </clr-date-container>
+      <clr-radio-container>
+        <label>Basic radio</label>
+        <clr-radio-wrapper>
+          <input type="radio" clrRadio name="radio" value="radio1" formControlName="radio" />
+          <label>Option 1</label>
+        </clr-radio-wrapper>
+        <clr-radio-wrapper>
+          <input type="radio" clrRadio name="radio" value="radio2" formControlName="radio" />
+          <label>Option 2</label>
+        </clr-radio-wrapper>
+        <clr-control-helper>Helper Subtext</clr-control-helper>
+        <clr-control-error>State Subtext</clr-control-error>
+        <clr-control-success>State Subtext</clr-control-success>
+      </clr-radio-container>
+      <clr-toggle-container>
+        <label>Full toggle switch</label>
+        <clr-toggle-wrapper>
+          <input type="checkbox" clrToggle value="option1" name="toggle-full" formControlName="toggle" />
+          <label>Option 1</label>
+        </clr-toggle-wrapper>
+        <clr-toggle-wrapper>
+          <input type="checkbox" clrToggle value="option2" name="toggle-full" formControlName="toggle" />
+          <label>Option 2</label>
+        </clr-toggle-wrapper>
+        <clr-control-helper>Helper Subtext</clr-control-helper>
+        <clr-control-error>State Subtext</clr-control-error>
+        <clr-control-success>State Subtext</clr-control-success>
+      </clr-toggle-container>
+    </form>
+  `,
+})
+class FormsStoryComponent {
+  _isDisabled = false;
+  _isSuccess = false;
+  _isError = false;
+
+  form = new FormGroup({
+    name: new FormControl(),
+    age: new FormControl(),
+    password: new FormControl(),
+    description: new FormControl(),
+    selectedOption: new FormControl(),
+    option1: new FormControl(),
+    date: new FormControl(),
+    radio: new FormControl(),
+    toggle: new FormControl(),
+  });
+
+  constructor(private changeDetectorRef: ChangeDetectorRef) {}
+
+  @Input()
+  get isDisabled() {
+    return this._isDisabled;
+  }
+  set isDisabled(value: boolean) {
+    this._isDisabled = value;
+    this.setControlsState();
+  }
+  @Input()
+  get isError() {
+    return this._isError;
+  }
+  set isError(value: boolean) {
+    this._isError = value;
+    this.setControlsState();
+  }
+  @Input()
```

Comment:
missing blank line here

Reply:
Resolved in [91a30d2](https://github.com/vmware-clarity/ng-clarity/pull/1461/commits/91a30d2499788a1e534b5c73cc51597c24f6c705)